osu.Game.Rulesets.Mania/Edit/Setup/ManiaDifficultySection.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Resources.Localisation.Web;
+using osu.Game.Screens.Edit.Setup;
+
+namespace osu.Game.Rulesets.Mania.Edit.Setup
+{
+    public partial class ManiaDifficultySection : DifficultySection
+    {
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            CircleSizeSlider.Label = BeatmapsetsStrings.ShowStatsCsMania;
+            CircleSizeSlider.Description = "The number of columns in the beatmap";
```

Comment:
Not something to address in this pull, but the "fun" thing with this is that the base `DifficultySection` has localisable descriptions, but we can't actually do that in ruleset DLLs right now... Going to need tooling support.

Reply:
Yep.